### DISCUSSION TITLE
BayBE 0.8.x and Python 3.8

### DISCUSSION THREAD

**Discussion by Scienfitz** [emdgroup/baybe#169] [Announcements]
We are discontinuing support for Python 3.8 due to major dependencies no longer supporting it.

- BayBE version `0.8.0` and `0.7.4` are functionally almost identical, if you need Python 3.8 you can install `0.7.4` but will no longer receive future updates. 

- Please be aware that pip might install you `0.7.4` and not the latest version if you run it from a Python 3.8 environment.

Soon we are looking to enable support for Python 3.12.

**Comment by Scienfitz** [comment]
Update: Python 3.12 is enabled since `baybe>=0.8.2`

**Comment by AVHopp** [comment]
Closing this as there is no need to "discuss" anything here and since this is quite outdated.

**Comment by Scienfitz** [comment]
Announcements don't have to be
- discussed
- closed
